### ISSUE TITLE
Datenschutzbeauftragter vs. Datenschutzverantwortlicher

### ISSUE THREAD

**Issue by riess82** [krabina/Datencockpit#9]
Zur leichteren Nachverfolgung auch hier dokumentiert:

Die meisten Verwender des Datencockpits werden keinen Datenschutzbeauftragten benötigen und damit auch keinen solchen freiwillig bestellen wollen. Laut WKO sollte die Person dann besser Datenschutzverantwortlicher heißen, um die Rechte und Pflichten klar abzugrenzen.

Im Datencockpit sollten diese 2 Bezeichnungen auswählbar sein.

**Comment by krabina** [comment]
Ja, das stimmt grundsätzlich. Im Datencockpit kann man durchaus bei einem Benutzer jederzeit "Datenschutzverantwortlicher" oder "Datenschutzverantwortliche/r" als Funktion eingeben. Einfach Tippen und auf Enter drücken. 

Unterschied ist, dass in den Stammdaten dann jeweils der Hinweis bleibt, das kein "Datenschutzbeauftrate/r" defniert wurde. Ich denke aber, dass es durchaus sinnvoll ist, das so zu lassen. Somit würde man auf einem Ausdruck des Verfahrensverzeichnisses der Behörde gegenüber dokumentieren, dass man eben keinen hat.

Ev. wäre es sinnvoll, eine weitere Funktion zu normieren, z. b. einen "Datenschutz-Kontaktperson", die dann schon überall ausgewiesen wird, aber ohne die offizielle Bezeichnung "beauftragter" zu verwende. "verantworlticher" halte ich da übrigens für Verwirrend, besser wäre da eben z. B. Kontaktperson, das ist von von der Begrifflichkeit offenbar unverfänglich und kann nicht mit dem -beauftragten verwechselt werden.
